styles/atom-import-cost.less

```diff
@@ -7,5 +7,5 @@
   opacity: 0.5;
   margin-left: 10px;
   position: relative;
-  top: -(@component-line-height - (@component-padding/2));
+  top: -@component-line-height;
```

Comment:
I also use a custom font. I just switched to the default font and it looks properly aligned.